0001-kaa-protocol/README.md

```diff
@@ -7,83 +7,96 @@ contributors: Alexey Gamov <agamov@cybervisiontech.com>
 ---
 
 ## Introduction
-
-The document describes general guidelines, principles, and requirements for **Kaa protocol**.
+The document describes the general requirements, principles, design and guidelines for the **Kaa Protocol** (KP).
+KP is a standard protocol designed for connecting clients and endpoints (colloquially - devices) to a Kaa server.
 
 ## Requirements and constraints
-
 - Clients must be able to connect to Kaa server without using an SDK and with minimum custom implementation.
-- Encrypted and unencrypted channels should be supported.
-- Protocol should work over MQTT and CoAP.
-- Protocol must be extensible to support future extensions.
+- Encrypted and unencrypted channels must be supported.
```

Comment:
I'll add the requirement for async messaging.
I'm not that sure about the resource path changes you propose. I'll let @vtkhir and @rasendubi comment if they please.